rf-notebook/src/main/docker/Dockerfile

```diff
@@ -14,20 +14,24 @@ EXPOSE 4040 4041 4042 4043 4044
 RUN conda install --quiet --yes \
     anaconda sphinx nbsphinx shapely numpy folium geopandas geojsonio
 
+RUN apt-get update && \
+    apt-get install -y gdal-bin && \
+    apt autoremove && \
+    apt-get clean all
```

Comment:
Attempt at using a pre-built GDAL. Unfortunately installs version 2.2.3, so more work is needed.